lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -90,10 +90,10 @@ public function execute(InputInterface $input, OutputInterface $output)
         $toSchema = $this->getSchemaProvider()->createSchema();
 
         //Not using value from options, because filters can be set from config.yml
-        if ( ! $isDbalOld && $filterExpr = $conn->getConfiguration()->getFilterSchemaAssetsExpression()) {
+        if (!$isDbalOld && $filterExpr = $conn->getConfiguration()->getFilterSchemaAssetsExpression()) {
```

Comment:
Given you are the lead of the project, I think the decision is yours, but you can probably discuss it with other doctrine core devs:

- either follow the PSR-2 rule
- or follow @guilhermeblanco's preference (or maybe @beberlei's one, I don't remember) which is part of the Doctrine coding standards (this is the only place where they are incompatible with PSR-2).

We just need to be consistent inside the package.

For me, it is a :+1: to be PSR-2 compatible (I've been asking for this for years, approximately since PSR-2 went out).